.github/release-initial-checklist.md

```diff
@@ -1,18 +1,22 @@
 The release pull request has been created! This checklist is a guide to follow for the remainder of the release process. You can check off each item in this list once completed.
 
-## Checklist
-
 * [ ] Checkout the release branch locally.
-* [ ] Create a GitHub milestone for the next release (set the start date as today's date).
-* [ ] If there are any remaining open issues in the milestone for {{version}}, do a final check with the team to verify none of those *need* to go into the release. After verifying, go through and move any open issues/pulls into the next milestone (except for any that have the `type: blocker` label). After this any `type: blocker` labelled issues must be complete and merged into the release branch (cherry-pick if necessary) before continuing.
+
+## Initial Prep - changelog and meta data changes
+
 * [ ] Copy the changelog from the pull request description into the `readme.txt` file in the changelog section. Create a new section here for this release, eg.g. `= {{version}} - 2020-01-20 =`.
 * [ ] Make any other changes to plugin metadata as necessary (no version changes needed, that's handled by script later in the process).
   * [ ] `readme.txt` - support versions changing, reference new blocks if necessary.
   * [ ] `woocommerce-gutenberg-products-block.php` - requirements/tested up to versions etc.
 * [ ] Push above changes to the release branch.
+
+## Testing Notes and Testing
+
+When creating testing notes, please write them from the perspective of a "user" (merchant) familiar with WooCommerce. So you don't have to spell out exact steps for common setup scenarios (eg. "Create a product"), but do be specific about the thing being tested. Include screenshots demonstrating expectations where that will be helpful.
+
 * [ ] Create testing notes for the release. You can usually go through the pull requests linked in the changelog and grab testing notes from each pull. Add the notes to `docs/testing/releases` and update the `docs/testing/releases/README.md` index.
+  * Note, make sure to differentiate between things in the testing notes that only apply to the feature plugin and things that apply when included in WooCommerce core as there may be variations there.
```

Comment:
(Note - I'm commenting on the docs/checklists as a whole, not just the diff. Some of my comments apply to unchanged lines - apologies for the confusion!)

Regarding the release QA testing step (`Smoke test built release zip using the testing instructions you created:`), I think we should remove the mention of `Smoke test` and the sub-items illustrating how to smoke test. 

This testing step is about QA/acceptance testing (as far as I understand); i.e. go through the testing instructions and make sure each new feature and bug is working as expected. This is different to smoke testing, which is quick (constant time, no matter how large the release), and typically done as the last step after release (to confirm the thing you released is what you intended).

Note I'm not advocating for _removing_ the smoke testing step(s) elsewhere  - e.g. just before deploy, and most importantly after deploy. Those are essential steps in my view. (Side note - the "how to smoke test" sub items could be moved there if they're useful.) The point is that the QA step is much more detailed testing, and could take longer / more team involvement depending on the size/risk of the release (e.g. big release - lots of QA testing).

Reply:
In the checklists, smoke test is included in only two places. You mentioned this:

> Note I'm not advocating for removing the smoke testing step(s) elsewhere - e.g. just before deploy, and most importantly after deploy.

Currently that's exactly where smoke testing is mentioned.